packages/opentelemetry-exporter-jaeger/src/jaeger.ts

```diff
@@ -48,6 +50,16 @@ export class JaegerExporter implements SpanExporter {
       tags: jaegerTypes.ThriftUtils.getThriftTags(tags),
     };
     this._sender.setProcess(this._process);
+
+    const flushInterval = config.flushInterval || 5000;
+    this._timer = setInterval(() => {
+      this._sender.flush((numSpans: number, err?: string) => {
+        if (err) {
+          this._logger.error(`failed to flush ${numSpans} spans: ${err}`);
+        }
+      });
+    }, flushInterval);
```

Comment:
Can we just `unref()` ? Having to call shutdown to clearInterval can be a pain... I think we will avoid a lot of github issues (because there is an active handle on the nodejs process)... And it will easier to integrate when people use third party packages like gulp etc.

Reply:
yeah sure